ports/stm32/eth.c

```diff
@@ -146,7 +129,7 @@ eth_t eth_instance;
 STATIC void eth_mac_deinit(eth_t *self);
 STATIC void eth_process_frame(eth_t *self, size_t len, const uint8_t *buf);
 
-STATIC void eth_phy_write(uint32_t phy_addr, uint32_t reg, uint32_t val) {
+void eth_phy_write(uint32_t phy_addr, uint32_t reg, uint32_t val) {
```

Comment:
Does it make sense to move this function and `eth_phy_read()` to the new `eth_phy.c` file?

Reply:
I considered that forth and back and decided to leave it in eth.c, because:
- eth_phy.c implements only the writes to registers that are different between the PHY devices. All common register accesses are still in eth.c.
- eth_phy_read() and eth_phy_write() access directly the MAC controller, like all the other code in eth.c. They are not specific for a different type of PHY device.

Reply:
OK.